zebra/zserv.c

```diff
@@ -2817,6 +2817,28 @@ DEFUN (no_ip_forwarding,
   return CMD_SUCCESS;
 }
 
+DEFUN (show_zebra,
+       show_zebra_cmd,
+       "show zebra",
+       SHOW_STR
+       "Zebra information\n")
+{
+  struct vrf *vrf;
+
+  vty_out (vty, "                            Route      Route      Neighbor   LSP        LSP%s", VTY_NEWLINE);
+  vty_out (vty, "VRF                         Installs   Removals    Updates   Installs   Removals%s", VTY_NEWLINE);
+  RB_FOREACH (vrf, vrf_name_head, &vrfs_by_name)
+    {
+      struct zebra_vrf *zvrf = vrf->info;
```

Comment:
I see that the pattern in the rest of zebra is to always check if `vrf->info` is NULL or not. But from my analysis this can never happen, so this is ok.